Containers/mastercontainer/Dockerfile

```diff
@@ -4,8 +4,8 @@ FROM docker:20.10.21-dind-alpine3.16 as dind
 # Caddy is a requirement
 FROM caddy:2.6.2-alpine as caddy
 
-# From https://github.com/docker-library/php/blob/master/8.0/bullseye/apache/Dockerfile
-FROM php:8.1.13-apache-bullseye
+# From https://github.com/docker-library/php/blob/master/8.1/alpine3.17/fpm/Dockerfile
+FROM php:8.1.13-fpm-alpine3.17
```

Comment:
lets wait for alpine 3.17.1 then as well

Reply:
Current Alpine Version 3.17.1 (Released Jan 09, 2023)

Reply:
Thanks for reminding for this, but it seems to be not available yeet on docker hub: https://hub.docker.com/_/alpine/tags?name=3.17.1
And even then it could take a while until the PHP repo updates to alpine v3.17.1

Reply:
the php image was now pushed to 3.17.1

Reply:
It looks like I will not get to this due to time problems before the next beta release which is planned for coming tuesday. Sorry for that! I'll try to include it in the version afterwards.